modules/TypeDefinitions.js

```diff
@@ -28,6 +28,13 @@ export type RouteProps = {
 }
 
 /**
+ * Style
+ */
+
+export type Styles = {[key: string]: Object};
+export type StyleSheet = Styles | Array<Styles>;
```

Comment:
Why not use this https://github.com/facebook/react-native/blob/9ee815f6b52e0c2417c04e5a05e1e31df26daed2/Libraries/StyleSheet/StyleSheetTypes.js#L15 ?